### DISCUSSION TITLE
Retrieving parent's next sibling?

### DISCUSSION THREAD

**Discussion by aaulia** [skrapeit/skrape.it#188] [Q&A] [answered]
I'm new to scrape, I can't seem to find how to retrieve text from parent's next sibling. So for example I have this HTML

```
<p style="margin-bottom: 5px">
    <strong>Producer</strong>
    :
</p>
<p>
      Foo, Bar
</p>
```

So I'm trying to reach `Foo, Bar` by first finding the `strong` element, go up to its parent, and then went down to the next sibling. How can I do this in skrape?

**Comment by christian-draeger** [comment, marked as answer]
hey,
assuming you have a html snippet like this:
```html
<p style="margin-bottom: 5px">
    <strong>Producer</strong>
    :
</p>
<p>
      Foo, Bar
</p>
```


you can do as follows to get the first sibling of the strongs parent element:
```kotlin
val firstSiblingOfStrongParent = htmlDocument(someHtmlSnippet) {
    strong {
        findFirst {
            parent {
                siblings[0]
            }
        }
    }
}

println(firstSiblingOfStrongParent.text) // will print:  Foo, Bar
```

with that
